subscriptions/management/commands/fix_subscription_lifecycle.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Fast forward subscriptions lifecycle up to given date. This "
+            "is used when the messages sending failed to get the subscription "
+            "up to date. Leave end_date blank for current date")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--end_date", dest="end_date", default=datetime.now(),
+            type=lambda today: datetime.strptime(today, '%Y%m%d'),
+            help='''Fast forward subscription to end_date
+                  By default it will use datetime.now() (format YYYYMMDD)'''
```

Comment:
I suspect this'll render all the blank space when using `--help`, if so rewriting it as single strings between brackets will fix it:
```python
help=('Fast forward subscription to end_date'
      'By default it will use datetime.now() (format YYYYMMDD)')
```